build.gradle

```diff
@@ -17,9 +14,14 @@ subprojects {
 
     apply plugin: "checkstyle"
     apply plugin: "pmd"
-    apply plugin: "com.github.spotbugs"
     apply plugin: "codenarc"
 
+    pmd {
+        toolVersion = "6.13.0"
+        ruleSetFiles = rootProject.files("config/pmd/ruleset.xml")
+        ruleSets = []
```

Comment:
@kjarosh Uwaga, to jest mocne. On przyjmuje to `ruleSetFiles`, ale równocześnie ma defaultowo coś w `ruleSets`